### PR TITLE
restyled Soldier Class Feats

### PR DESCRIPTION
**Pull request by bleehu**
and removed Battle Adrenaline, added Battle Hardened and Grizzled Veteran.
Simpler to play, and adds tanky-ness to the Soldier class in accordance
with design docs. Also removes another instance of old-instance nanites.

### CODE REVIEW

File: CharacterCreation/02_Classes.txt [hunk 1]
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
...
-	V
-	The soldier's accuracy bonus effect works on any enemy with a lower initiative until
-	that enemy has finished their 3rd turn.
-======================================
+
+Battle Hardened:
+
+    Your body and mind are used to the constant strain of combat. You no longer
+recoil quite so much from the horrors of war like a greenhorn and it takes more
+to shock you. Your pain tolerance has grown and you are more concious of your
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: sp: conscious.

File: CharacterCreation/02_Classes.txt [hunk 2]
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
...
+* + 8 to your Reflex Save
+* + 8 to your Awareness Save
+* + 30 to your maximum health
+
+Grizzled Veteran:
+
+    You thrive on gunfighting. You love the smell of napalm in the morning. 
+You've had so much shrappnel in your body that it's gotten homeostatically used
+to it. This Feat cannot be taken unless you have "Battle Hardened" and are level 
+10. You gain the following.
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: "following:"

File: CharacterCreation/02_Classes.txt [hunk 3]
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
...
-	The soldier's accuracy bonus effect works on any enemy with a lower initiative until
-	that enemy has finished their 3rd turn.
-======================================
+
+Battle Hardened:
+
+    Your body and mind are used to the constant strain of combat. You no longer
+recoil quite so much from the horrors of war like a greenhorn and it takes more
+to shock you. Your pain tolerance has grown and you are more concious of your
+surroundings. You get the following:
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: "gain" (consistency)

File: CharacterCreation/02_Classes.txt [hunk 4]
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
...
+Raise Morale.
+
+Fight Fiercely First:
+
+	You get an additional +10 to initiative rolls. In addition, you get a -3 
+miss chance against enemies who have not taken a turn yet in combat or whom you
+have beaten in the reflex roll when room clearing until that character takes a 
+turn.
+
+Surpise and Violence of Action:
```
Comment (Turtlelord26) [inline CharacterCreation/02_Classes.txt:96]: sp: Surprise

**Review by Turtlelord26 — APPROVED** [review]
Remind me to merge these into my nanite rework superbranch.

File: CharacterCreation/02_Classes.txt [hunk 5]
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
...
+You've had so much shrappnel in your body that it's gotten homeostatically used
+to it. This Feat cannot be taken unless you have "Battle Hardened" and are level 
+10. You gain the following.
+
+* An additional +20 to your maximum health
+* If you would fail a reload roll, you succeed instead    
+
+Raise Morale:
+
+	You roar to incite courage in your allies. 1 Action. 50% + (CHA mod) chance
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: I think this is equivalent to making a DC 50 Cha check? If so, I think it's better to phase the ability in a way that is consistent with how we usually describe making rolls.

File: CharacterCreation/02_Classes.txt [hunk 6]
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
...
+
+	You roar to incite courage in your allies. 1 Action. 50% + (CHA mod) chance 
+to give allied Characters a +20 on their next skill roll or +2 on their next
+attack roll. This includes your next roll. Allies must be able to hear and 
+understand you to get the bonus. Max 1 successful use per encounter.
+
+Battle Cry:
+
+	You give a violent yell that spurs you and your teammates on, giving them
+the sense that victory is possible. DC 25 - CHA mod to be successful. If
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: Same comment as above: DC 25 Cha check

File: CharacterCreation/02_Classes.txt [hunk 7]
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
...
+understand you to get the bonus. Max 1 successful use per encounter.
+
+Battle Cry:
+
+	You give a violent yell that spurs you and your teammates on, giving them
+the sense that victory is possible. DC 25 - CHA mod to be successful. If 
+successful, your Battle Cry gives you and your teammates a +30 on their next 
+skill roll or +3 added to their next attack roll. A Character must be level 10 
+and have the "Raise Morale" Class Feat in order to take this feat. This ability
+takes 1 Action andcan only be used once per encounter. This ability replaces
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: "and can"

File: CharacterCreation/02_Classes.txt [hunk 8]
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
...
+the sense that victory is possible. DC 25 - CHA mod to be successful. If 
+successful, your Battle Cry gives you and your teammates a +30 on their next 
+skill roll or +3 added to their next attack roll. A Character must be level 10 
+and have the "Raise Morale" Class Feat in order to take this feat. This ability
+takes 1 Action andcan only be used once per encounter. This ability replaces 
+Raise Morale.
+
+Fight Fiercely First:
+
+	You get an additional +10 to initiative rolls. In addition, you get a -3
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: "gain"

File: CharacterCreation/02_Classes.txt [hunk 9]
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
...
+successful, your Battle Cry gives you and your teammates a +30 on their next 
+skill roll or +3 added to their next attack roll. A Character must be level 10 
+and have the "Raise Morale" Class Feat in order to take this feat. This ability
+takes 1 Action andcan only be used once per encounter. This ability replaces 
+Raise Morale.
+
+Fight Fiercely First:
+
+	You get an additional +10 to initiative rolls. In addition, you get a -3 
+miss chance against enemies who have not taken a turn yet in combat or whom you
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: "+3 accuracy"

File: CharacterCreation/02_Classes.txt [hunk 10]
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
...
+have beaten in the reflex roll when room clearing until that character takes a 
+turn.
+
+Surpise and Violence of Action:
+
+	The soldier's accuracy bonus effect works on any enemy with a lower 
+initiative until that enemy has finished their 3rd turn. A Character cannot take
+this Class Feat unless they are level 10 and have the "Fight Fiercely First"
+Class Feat as well.
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt:101]: delete "as well."

**Review by rosebrallier — CHANGES_REQUESTED** [review]
XD
I know you didn't set out to change Raise Morale much, but I have suggestions (and concerns about how the original was written)

File: CharacterCreation/02_Classes.txt (hunk 5 above)
Comment (Turtlelord26) [reply CharacterCreation/02_Classes.txt]: Confirm mathematically identical.

**Comment by bleehu** [comment]
Dorp. My bad.

**Comment by bleehu** [comment]
Ya, I've got some hesitation about it too. I think it originated as a way to try to force charisma to have combat relevance. Could you capture your concerns either here, in CX design, or on slack? Whichever works best for you.

**Comment by rosebrallier** [comment]
My concerns are more from a "writing a comprehensible thing" perspective than a gameplay perspective.

File: CharacterCreation/02_Classes.txt [hunk 11]
```diff
@@ -44,40 +44,61 @@ Skill Proficiencies:
...
+
+Grizzled Veteran:
+
+    You thrive on gunfighting. You love the smell of napalm in the morning. 
+You've had so much shrappnel in your body that it's gotten homeostatically used
+to it. This Feat cannot be taken unless you have "Battle Hardened" and are level 
+10. You gain the following:
+
+* An additional +20 to your maximum health
+* If you would fail a reload roll, you succeed instead
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: Maybe we should exclude crit fails from this?

**Review by rosebrallier — APPROVED** [review]
Ship-it! I left one suggestion re allowing failure, but take it or leave it as you see fit.

**Pull request merged**